.gitignore

```diff
@@ -163,4 +163,6 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
-# End of https://www.toptal.com/developers/gitignore/api/python
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/python
+
+.idea/
```

Comment:
This is a Toptal templated `.gitignore`, it is preferred to use the edit link at the top of this file to make edits instead of one off changes.

That said, IDE ignores don't belong in the repo, otherwise we'll end up needing to ignore stuff from every IDE in existence. It's usually better for you to add such ignores to your personal global `.gitignore`, see https://gist.github.com/subfuzion/db7f57fff2fb6998a16c for more details.

Reply:
I'll just remove the `.gitignore` file and let people figure that out themselves.